pkg/ccl/backupccl/backupinfo/manifest_handling.go

```diff
@@ -82,15 +83,18 @@ const (
 
 // WriteMetadataSST controls if we write the experimental new format BACKUP
 // metadata file.
-// kv.bulkio.rite_metadata_sst.enabled set to false by default due to
-// https://github.com/cockroachdb/cockroach/issues/85564.
 var WriteMetadataSST = settings.RegisterBoolSetting(
 	settings.TenantWritable,
 	"kv.bulkio.write_metadata_sst.enabled",
 	"write experimental new format BACKUP metadata file",
-	true,
+	false,
```

Comment:
Rather than adding a conditional to the test above, any reason not to put the ConstantWithMetamorphicTestBool right here?

Reply:
heh, good point. done.